### DISCUSSION TITLE
How to proxy a service that is running on the docker host?

### DISCUSSION THREAD

**Discussion by itsmaty** [NginxProxyManager/nginx-proxy-manager#3874] [Q&A]
My current setup only proxies services also running in docker, to proxy them i add them all together with npm in an docker network and in npm i refer to the service by the container name and it works just fine.

However, how can i proxy a service that is running on the docker host (so directly on my machine).
How can i "break out" of docker and refer to the host machine from inside a docker container?

**Comment by pkb-code** [comment]
You can use the host's IP in "Forward Hostname / IP" field :)
